build.sbt

```diff
@@ -129,11 +130,20 @@ lazy val openlawCore = crossProject(JSPlatform, JVMPlatform)
   .settings(releaseSettings: _*)
   .enablePlugins(WartRemover)
 
+
+lazy val version = "sbt version"!
```

Comment:
I'm wondering if there is any way we can avoid shelling out to get this info (which can break in unexpected ways due to $PATH issues, etc). Does SBT not have access to this info in a local variable already?

Reply:
Looking at the sbt-git source code, I think we _may_ be able to get this via `git.gitDescribedVersion`? https://github.com/sbt/sbt-git/blob/master/src/main/scala/com/typesafe/sbt/SbtGit.scala#L120

Reply:
done!